operator/controllers/operation_controller.go

```diff
@@ -24,13 +24,14 @@ import (
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
I think import from github.com should go in the first group (they are not k8s.io/sigs-k8s.io)
@yastij opinions ^^^ ?

Reply:
@fabriziopandini  btw goimports does not allow grouping std imports with third-party ones, so I guess these imports have to go to the 2nd group

Reply:
BTW we should be following what kubernetes/kubernetes is doing for go formatting.
the tool for that is `go fmt` and not `goimports`:
https://github.com/kubernetes/kubernetes/blob/master/hack/update-gofmt.sh

Reply:
@neolit123 If so, what's the decision here? Do we then even need to use 3 groups of imports, if there is no agreement in k/k on this, and if we can't force this via automation?

Reply:
this repo already has CI with gofmt enabled, so it will catch problems. but i think it's still possible to use multiple groups.